doc/guides/general/vision.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: The Vision of RIOT
+description: The vision of RIOT
```

Comment:
If this is a copy of the current [vision page](https://doc.riot-os.org/vision.html), that should be removed. If you do this for all migrated guides, this may also give you hints on what changed since you started this migration (see also https://github.com/RIOT-OS/RIOT/pull/21404#discussion_r2040642734)

Reply:
Fixed